docker/lr-utils/Dockerfile

```diff
@@ -13,6 +13,63 @@ RUN echo "source activate lr-utils" > ~/.bashrc
 ENV PATH=/opt/conda/envs/lr-utils/bin/:/root/google-cloud-sdk/bin/:${PATH}
 
 # install gsutil
-RUN apt update
+RUN apt-get --allow-releaseinfo-change update
 RUN apt install -y curl git-lfs parallel
 RUN curl https://sdk.cloud.google.com | bash
+
+# Setup crcmodc for gsutil:
+RUN apt-get install -y gcc python3-dev python3-setuptools && \
+			pip3 uninstall -y crcmod && \
+			pip3 install --no-cache-dir -U crcmod
+
+# Install bamtools:
+# First get cmake:
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.20.1/cmake-3.20.1-linux-x86_64.tar.gz \
+		&& tar -zxf cmake-3.20.1-linux-x86_64.tar.gz \
+		&& mv cmake-3.20.1-linux-x86_64/bin/* /usr/local/bin/ \
+		&& mv cmake-3.20.1-linux-x86_64/doc/* /usr/local/doc/ \
+		&& mv cmake-3.20.1-linux-x86_64/man/* /usr/local/man/ \
+		&& mv cmake-3.20.1-linux-x86_64/share/* /usr/local/share/ \
+		&& rm -rf cmake-3.20.1-linux-x86_64 cmake-3.20.1-linux-x86_64.tar.gz
+
+# Get g++
+RUN apt-get install -y g++
+RUN apt-get install -y make
+RUN apt-get install -y zlib1g zlib1g-dev
+
+RUN git clone git://github.com/pezmaster31/bamtools.git \
+    && cd bamtools \
+    && mkdir build && cd build \
+    && cmake .. \
+    && make \
+    && make install \
+    && cd ../../ \
+    && rm -rf bamtools
+
+RUN wget https://github.com/broadinstitute/picard/releases/download/2.26.10/picard.jar \
+    && mkdir -vp /usr/picard \
+    && mv picard.jar /usr/picard/ \
+    && echo -e '#!/usr/bin/env bash\njava -jar /usr/picard/picard.jar $@' > picard.sh \
+    && chmod +x picard.sh
+
+
+# Install samtools 1.11:
+#RUN apt-get update
+RUN apt-get install -y bzip2 curl gnupg2
+RUN apt-get install -y libc-dev ncurses-dev
+
+#### Specific for google cloud support
```

Comment:
Can you provide reasons why these tools are needed specifically in `lr-utils`?
Utils in general tend to be dumping grounds. I'm trying hard to avoid that here.

Reply:
I think I added these in to satisfy some requirements in samtools for extended features.